roles/nova-data/tasks/monitoring.yml

```diff
@@ -13,6 +13,7 @@
   sensu_metrics_check: name=check-cinder-sessions plugin=check-cinder-sessions.py
                        interval=10
   notify: restart sensu-client
+  when: "'rbd_volumes' not in cinder.enabled_backends"
```

Comment:
is enabled_backends always a list?

`when: not cinder.enabled_backends|default('')|search('rdb_volumes')` ?